libraries/fiff/fiff_evoked_set.h

```diff
@@ -193,12 +193,14 @@ class FIFFSHARED_EXPORT FiffEvokedSet
 
 #ifndef metatype_fiffevokedset
 #define metatype_fiffevokedset
-Q_DECLARE_METATYPE(FIFFLIB::FiffEvoked); /**< Provides QT META type declaration of the FIFFLIB::FiffEvoked type. For signal/slot usage.*/
+Q_DECLARE_METATYPE(FIFFLIB::FiffEvoked);/**< Provides QT META type declaration of the FIFFLIB::FiffEvoked type. For signal/slot and QVariant usage.*/
+Q_DECLARE_METATYPE(FIFFLIB::FiffEvokedSet);/**< Provides QT META type declaration of the FIFFLIB::FiffEvokedSet type. For signal/slot and QVariant usage.*/
 #endif
 
 #ifndef metatype_fiffevokedsetsptr
 #define metatype_fiffevokedsetsptr
-Q_DECLARE_METATYPE(FIFFLIB::FiffEvoked::SPtr); /**< Provides QT META type declaration of the FIFFLIB::FiffEvoked type. For signal/slot usage.*/
+Q_DECLARE_METATYPE(FIFFLIB::FiffEvoked::SPtr);/**< Provides QT META type declaration of the FIFFLIB::FiffEvoked type. For signal/slot and QVariant usage.*/
```

Comment:
Although this was not introduced in your PR: FiffEvoked::SPtr should be moved to to fiffevoked.h